packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -118,7 +118,10 @@ describe("SequenceField - Rebaser Axioms", () => {
                 it(`${name} ○ ${name}⁻¹ === ε`, () => {
                     const change = [mark];
                     const inv = SF.invert(makeAnonChange(change), TestChange.invert);
-                    const actual = SF.compose([change, inv], TestChange.compose);
+                    const actual = SF.compose(
+                        [makeAnonChange(change), makeAnonChange(inv)],
```

Comment:
In this case we acutally want them to be tagged so they can properly cancell out.